gaia-header.js

```diff
@@ -1,29 +1,52 @@
 ;(function(define){'use strict';define(function(require,exports,module){
+/*jshint laxbreak:true*/
 /*jshint esnext:true*/
+/*jshint boss:true*/
 
 /**
  * Dependencies
  */
 
-var Component = require('gaia-component');
-var fontFit = require('./lib/font-fit');
+var component = require('gaia-component');
+var fontFit = require('font-fit');
 
-// Load 'gaia-icons' font-family
+/**
+ * Load 'gaia-icons' font-family
+ */
 require('gaia-icons');
 
 /**
+ * Simple logger (toggle 0)
+ *
+ * @type {Function}
+ */
+var debug = 0 ? console.log.bind(console) : function() {};
```

Comment:
because you use const then, maybe you can use it here as well? I don't think we change it during the course of the program